src/steps/relay.py

```diff
@@ -128,3 +159,18 @@ def create_message(self, **kwargs):
     def subscribe_and_publish_with_retry(self, node_list, pubsub_topic_list):
         self.ensure_relay_subscriptions_on_nodes(node_list, pubsub_topic_list)
         self.check_published_message_reaches_relay_peer()
+
+    @allure.step
+    def register_rln_single_node(self, **kwargs):
+        step_id = f"{datetime.now().strftime('%Y-%m-%d_%H-%M-%S')}__{str(uuid4())}"
```

Comment:
maybe we can add this as helper function in libs/common and use it here and in conftest

Reply:
Added at https://github.com/waku-org/waku-interop-tests/pull/16/commits/9a36ac52e75a4cd20b7a9f5e062d5dc3cc1597f2